docs/public-networks/get-started/connect/mainnet.md

```diff
@@ -0,0 +1,141 @@
+This is a shared JWT secret the clients use to authenticate each other when using the
+[Engine API](../../how-to/use-engine-api.md).
+
+### 2. Start Besu
+
+Run the following command or specify the options in a
+[configuration file](../../how-to/configuration-file.md):
+
+```bash
+besu \
+  --sync-mode=X_SNAP           \
+  --data-storage-format=BONSAI \
+  --rpc-http-enabled=true      \
+  --rpc-http-host=0.0.0.0      \
+  --rpc-ws-enabled=true        \
+  --rpc-ws-host=0.0.0.0        \
+  --host-allowlist=<IP of Besu node>,127.0.0.1,localhost        \
+  --engine-host-allowlist=<IP of Besu node>,127.0.0.1,localhost \
+  --Xmerge-support=true        \
+  --engine-jwt-secret=<path to jwtsecret.hex>
+```
+
+Specify:
+
+- The path to the `jwtsecret.hex` file generated in [step 1](#1-generate-the-shared-secret)
+  using the [`--engine-jwt-secret`](../../reference/cli/options.md#engine-jwt-secret) option.
+- The IP address of your Besu node using the [`--host-allowlist`](../../reference/cli/options.md#host-allowlist)
+  and [`--engine-host-allowlist`](../../reference/cli/options.md#engine-host-allowlist) options.
+
+This command also specifies the following options:
+
+- [`sync-mode`](../../reference/cli/options.md#sync-mode) - Use [snap sync](sync-node.md#snap-synchronization).
+- [`data-storage-format`](../../reference/cli/options.md#data-storage-format) - Use [Bonsai Tries](../../concepts/data-storage-formats.md#bonsai-tries).
+- [`rpc-http-enabled`](../../reference/cli/options.md#rpc-http-enabled) - Enable the HTTP JSON-RPC
+  service.
+- [`rpc-http-host`](../../reference/cli/options.md#rpc-http-host) - Set to `0.0.0.0` to allow remote
+  RPC connections.
+- [`rpc-ws-enabled`](../../reference/cli/options.md#rpc-ws-enabled) - Enable the WebSocket JSON-RPC
+  service.
+- [`rpc-ws-host`](../../reference/cli/options.md#rpc-ws-host) - Set to `0.0.0.0` to allow remote RPC
+  connections.
+- `Xmerge-support` - Enable Merge support.
+
+You can modify the option values and add other [command line options](../../reference/cli/options.md)
+as needed.
+
+### 3. Generate validator keys and stake ETH
+
+If you're running a beacon node only, skip to the next step.
+
+If you're also running a validator client, have a funded Ethereum address ready (32 ETH and gas fees
+for each validator).
+
+Generate validator keys and stake your ETH for one or more validators using the
+[Staking Launchpad](https://launchpad.ethereum.org/en/).
+
+!!! important
+
+    Save the password you use to generate each key pair in a `.txt` file.
+    You should also have a `.json` file for each validator key pair.
+
+    Ensure your Besu node is fully synced before submitting your staking deposit.
+    This can take several days.
+
+### 4. Start the consensus client
+
+Refer to your consensus client documentation to configure and start the consensus client.
+If you're running a validator client, make sure you set a fee recipient address.
```

Comment:
This could be made important as it seems some users forgot to set this. I found some blocks on mainnet with default recipient - 0x0000000000000000000000000000000000000000 - so may be worth highlighting the need to configure it.